meta-cube/recipes-support/overc-utils/source/cube-cfg

```diff
@@ -591,15 +591,20 @@ IFS='
 
     # Generate an oci.cmd that can be used to launch this container via runc
     echo "#!/bin/bash" > oci.cmd
-    echo "mgr=\$(cat /opt/container/${container_name}/cube.container.mgr)" >> oci.cmd
     if [ "${console_mgr}" = "tty" ]; then
-	runc_console="--console-socket=/dev/null"
+	runc_console="--console-socket=/var/lib/cube/$container_name/runc-sock"
+	echo "rm -f /var/lib/cube/${container_name}/dtach-sock /var/lib/cube/${container_name}/runc-sock" >> oci.cmd
+	echo "/usr/bin/dtach -n /var/lib/cube/${container_name}/dtach-sock -W 120 -p /var/lib/cube/${container_name}/pty-console -c -C -E -q -Q /var/lib/cube/${container_name}/runc-sock" >> oci.cmd
```

Comment:
As we were saying before, these could be hooks (pre-start here, post-start below), but I think that this is better for making the details easily accessible (at least for now) .. plus we really control the timing, versus counting on runc to not change.

If things get more complex we could revisit and move them to hooks.